epytope/EpitopeSelection/OptiTope.py

```diff
@@ -176,7 +199,7 @@ def __init__(self, results,  threshold=None, k=10, solver="glpk", verbosity=0):
         model.Q = Set(initialize=variations)
 
         model.E = Set(initialize=set(peps.keys()))
-
+        logging.warning(set(peps.keys()))
```

Comment:
```suggestion
```

Reply:
Should be imported at the top. I'll double check in the next commit 👍

Reply:
```suggestion
```